internal/plugins/golang/v2/init.go

```diff
@@ -34,20 +38,35 @@ var _ plugin.Init = &initPlugin{}
 func (p *initPlugin) UpdateContext(ctx *plugin.Context) { p.Init.UpdateContext(ctx) }
 func (p *initPlugin) BindFlags(fs *pflag.FlagSet)       { p.Init.BindFlags(fs) }
 
+// InjectConfig will inject the PROJECT file/config in the plugin
 func (p *initPlugin) InjectConfig(c *config.Config) {
 	p.Init.InjectConfig(c)
 	p.config = c
 }
 
+// Run will call the plugin actions
 func (p *initPlugin) Run() error {
+	return cmdutil.Run(p)
+}
+
+// Validate will perform the required checks before to start to scaffold
+func (p *initPlugin) Validate() error {
+	return nil
+}
+
+// GetScaffolder will run the plugin scaffold
+func (p *initPlugin) GetScaffolder() (scaffold.Scaffolder, error) {
```

Comment:
Same comment: why does this run the scaffolding?

Reply:
It is defined in the interface already. However, I will push an improvement to it.